cli/studio.py

```diff
@@ -143,6 +143,8 @@ def build_all_scenarios(clean, allow_offset_maps, scenarios):
         path = Path(scenarios_path)
         for p in path.rglob("*.net.xml"):
             scenario = f"{scenarios_path}/{p.parent.relative_to(scenarios_path)}"
+            if scenario == f"{scenarios_path}/waymo":
+                continue
```

Comment:
I'm not sure about this "short circuit" here.  In addition to wanting to avoid "special cases" like this in general, I think the danger is that `_build_single_scenario()` also (perhaps eventually) does something necessary aside from (mostly) Sumo stuff.  In fact, right now, it can be used to clean the scenario folder as well as install additional pip requirements.  Even though "_our_" Waymo use cases don't require these, other users' might.

Could we move this check further down the call stack and/or make it more generic?  (For example, lower down the stack, maybe we can skip things related to Sumo in general if there is no `map.net.xml` file.)

Reply:
currently we don't build any scenario with `map.net.xml` file missing. Here i skip building the waymo scenario since it tries to load in the Waymo dataset which is not added to the repo and hence throws the above error. The only we can stop it is to skip building the Waymo scenario during `build-all` command is by the above check or remove its `map.net.xml` file. So we decided to go with this one. If we move it further below the call stack it means we will be putting it in the  `_build_single_scenario()` where we don't want to since we still want to build it individually.

Reply:
After some discussion we decided to remove the `map.net.xml` since it for an external dataset that we don't have available available and makes the scenario incomplete (Like NGSIM). So this takes care of the above issue.